services/federation/followers.go

```diff
@@ -0,0 +1,36 @@
+package federation
+
+import (
+	"github.com/yitsushi/go-misskey/core"
+	"github.com/yitsushi/go-misskey/models"
+)
+
+// followersResponse contains a list of followers.
+type followersResponse struct {
+	Followers []models.Followers
+}
+
+// FollowersRequest contains request information for the followers call.
+type FollowersRequest struct {
+	Host    string `json:"host"`
+	SinceID string `json:"sinceId"`
+	UntilID string `json:"untilId"`
+	Limit   int    `json:"limit"`
+}
+
+// Validate the request.
+func (r *FollowersRequest) Validate() error {
+	return nil
+}
+
+// Followers lists all followers.
+func (s *Service) Followers(request *FollowersRequest) ([]models.Followers, error) {
```

Comment:
Argument as non-pointer, and pass it as pointer to JSONRequest. That way there is no way the function can mutate the variable outside of its scope and we don't have to check for `nil`.